src/applications/simple-forms/21-0966/config/submit-transformer.js

```diff
@@ -5,10 +5,42 @@ import {
 } from '../definitions/constants';
 
 export default function transformForSubmit(formConfig, form) {
-  const transformedData = JSON.parse(
+  const sharedTransformedData = JSON.parse(
     sharedTransformForSubmit(formConfig, form),
   );
 
+  // We need to transform the data this way because the prefill keys don't have the 'veteran'
+  // prefix. The backend expects 'veteranFullName' and not 'fullName', for example.
+  const transformedData = {
+    ...sharedTransformedData,
+    veteranFullName:
+      sharedTransformedData.veteranFullName || sharedTransformedData.fullName,
+    veteranDateOfBirth:
+      sharedTransformedData.veteranDateOfBirth ||
+      sharedTransformedData.dateOfBirth,
+    veteranId: {
+      ssn: sharedTransformedData.veteranId.ssn || sharedTransformedData.ssn,
```

Comment:
can we also add optional chaining to these? will just give me more peace knowing an exception won't be thrown here.
`sharedTransformedData.veteranId?.ssn` and `sharedTransformedData.veteranId?.vaFileNumber`